scrunch/tests/test_datasets.py

```diff
@@ -1997,7 +1997,6 @@ def prepare_ds(self):
         dataset_url = 'http://host/api/projects/abc/'
         variables_url = "http://host/api/projects/abc/variables/"
         new_var_url = "http://host/api/projects/abc/variables/123/"
-        hier_url = "http://host/api/projects/abc/variables/hier/"
```

Comment:
Now accessing this endpoint is lazy, so it is not a request done during initialization. It was here as part of the needed fixtures to init a dataset